src/mesh/meshblock.cpp

```diff
@@ -204,6 +204,7 @@ MeshBlock::~MeshBlock() = default;
 void MeshBlock::InitializeIndexShapesImpl(const int nx1, const int nx2, const int nx3,
                                           bool init_coarse, bool multilevel) {
   cellbounds = IndexShape(nx3, nx2, nx1, Globals::nghost);
+  f_cellbounds = IndexShape(2 * nx3, 2 * nx2, 2 * nx1, Globals::nghost);
```

Comment:
Does this translate to a fine block having the same number of ghost zones than a standard block?
If so, is that desired behavior as this might result in weird indexing wrt to fine ghost data (and similarly in the context of prolongation/restriction).

Reply:
Yes. I went with this implementation since it was analogous to the difference between a regular data and course data for a field. That way for a fine field, the course data should have the same shape as regular data and I think prolongation and restriction should work out of the box.